openwebstart/src/main/java/com/openwebstart/jvm/RemoteRuntimeManager.java

```diff
@@ -62,14 +61,9 @@ private RemoteRuntimeManager() {
                 .filter(c -> Objects.equals(endpointForRequest, c.getEndpointForRequest()))
                 .map(c -> (Result<RemoteRuntimeList>) new Sucess(c.getList()))
                 .orElseGet(Result.of(() -> {
-                    final HttpClient client = Client.getService(HttpClient.class);
-                    final HttpResponse<String> response = client.get(endpointForRequest)
-                            .withoutContent()
-                            .readString()
-                            .execute()
-                            .get(RuntimeManagerConstants.HTTP_TIMEOUT_IN_MS, TimeUnit.MILLISECONDS);
-
-                    final String jsonContent = response.getContent();
+                    final HttpRequest request = new HttpRequest(endpointForRequest);
+                    final HttpResponse response = request.handle();
+                    final String jsonContent = IOUtils.readContentAsString(response.getContentStream(), Charset.forName("UTF-8"));
```

Comment:
`Charset.forName("UTF-8")` can be replaced by `StandardCharsets.UTF_8` which does not throw an exception